### PR TITLE
added twig cs

### LINKED ISSUE #432: twig code style ?
https://github.com/VincentLanglet/Twig-CS-Fixer

we should investigate if we want to enforece a twig code style in phpstorm and in CI actions

### PR DESCRIPTION
**Pull request by bidi47**

### CODE REVIEW

File: config/.twig-cs-fixer.php [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+$config = new TwigCsFixer\Config\Config();
+$config->addTwigExtension(new Dot\Twig\Extension\DateExtension());
+$config->addTwigExtension(new Dot\Twig\Extension\TranslationExtension());
+
+return $config;
```
Comment (alexmerlin) [inline config/.twig-cs-fixer.php]:
```suggestion
return $config;

```

File: config/.twig-cs-fixer.php
Comment (alexmerlin) [inline config/.twig-cs-fixer.php:1]: File name should not start with `.`.
If you remove the dot from the file name, then make sure you modify in the composer script as well.

**Review by alexmerlin — CHANGES_REQUESTED** [review]

File: config/.twig-cs-fixer.php
Comment (bidi47) [reply config/.twig-cs-fixer.php:1]: resolved in more recent commit

File: composer.json [hunk 2]
```diff
@@ -116,6 +117,7 @@
         "serve": "php -S 0.0.0.0:8080 -t public/",
         "static-analysis": "psalm --shepherd --stats",
         "test": "phpunit --colors=always",
-        "test-coverage": "phpunit --colors=always --coverage-clover clover.xml"
+        "test-coverage": "phpunit --colors=always --coverage-clover clover.xml",
+        "twig-cs-fixer": "vendor/bin/twig-cs-fixer lint --config=config/twig-cs-fixer.php"
```
Comment (alexmerlin) [inline composer.json]:
```suggestion
        "twig-cs-check": "vendor/bin/twig-cs-fixer lint --config=config/twig-cs-fixer.php",
        "twig-cs-fix": "vendor/bin/twig-cs-fixer lint --config=config/twig-cs-fixer.php --fix"
```

This way we have a command for checking and another for fixing.

**Review by alexmerlin — APPROVED** [review]

**Comment by bidi47** [comment]
@arhimede, is this ok to push to the default branch?

**Pull request merged**